src/instructlab/model/evaluate.py

```diff
@@ -229,36 +234,44 @@ def launch_server(
     model_name,
     max_workers,
     gpus,
+    backend,
 ) -> tuple:
     # pylint: disable=import-outside-toplevel
     # First Party
-    from instructlab.model.backends import backends
 
-    if not ctx.obj.config.serve.backend:
-        ctx.obj.config.serve.backend = backends.VLLM
-    if ctx.obj.config.serve.backend == backends.VLLM:
-        ctx.obj.config.serve.vllm.vllm_args.extend(["--served-model-name", model_name])
+    # we shouldn't be in the business of editing a user's cfg.
+    # with eval, this is tricky since there are a few things we want to help them out with if they do not exist.
```

Comment:
It's not really about not existing and filling in blanks or overriding user config.  The model is coming from an option or the eval config.  It's just utilizing serve config to not have to duplicate it for every setting.

Reply:
deleting this for now, but once we wire up the serve cfg into this command more I might add some notice like this back in